gfx/NeoDays/pngs_tiles_10x10/items/minireactor.json

```
@@ -0,0 +1,4 @@
+{
+  "id": "minireactor",
```

```suggestion
  "id": ["minireactor", "vp_minireactor"],
```
otherwise vp_minireactor will only be used for the item and not for the vehicle part

---

There is a separate json file for the vehicle minireactor under *10x10/vehicle/